iris_grib/tests/unit/test_save_messages.py

```diff
@@ -23,22 +23,14 @@ def setUp(self):
     def test_save(self):
         m = mock.mock_open()
         with mock.patch('builtins.open', m, create=True):
-            # sending a MagicMock object to gribapi raises an AssertionError
-            # as the gribapi code does a type check
-            # this is deemed acceptable within the scope of this unit test
-            with self.assertRaises((AssertionError, TypeError)):
```

Comment:
The `AssertionError` and `TypeError` no longer get raised in the latest version of eccodes